components/nodemanager-service/nodemanager.go

```diff
@@ -57,6 +63,32 @@ func Serve(conf config.Nodemanager, grpcBinding string) error {
 	return serve(ctx, &conf, connFactory)
 }
 
+const (
+	Awsec2PollingJobName        = "awsec2_polling"
```

Comment:
[nit] these constants... look funny? I'm not sure I like the alternatives better though. Perhaps `AWSEC2PollingJobName` or `AwsEc2PollingJobName`?